docs/modules/ROOT/pages/ios_task_scheduling.adoc

```diff
@@ -0,0 +1,33 @@
+= Task Scheduling
+
+Background tasks are scheduled based on the app's current context, and that context is based on a combination of factors. 
+These factors include:
+
+* Is the app backgrounded or in the foreground?
+* Is WiFi available?
+* Is the device in Low Power Mode?
+* Is the device connected to external power?
+* Has the photo library has changed?
+
+== Tasks 
+
+Currently, two tasks are available:
+
+* *Instant photo uploads:* This task is triggered when:
```

Comment:
Is this "photo and video"?
If so, then "The photo library has new photos" will need enhancement also.

Reply:
Thanks for spotting that.